include/deal.II/fe/mapping_fe.h

```diff
@@ -0,0 +1,741 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2001 - 2015 by the deal.II authors
```

Comment:
Oh well, we didn't start from scratch... I'd say 90% of it is based on code
from the early days...

:-)

Luca

On 06/apr/2015, at 15:28, Wolfgang Bangerth <notifications@github.com>
wrote:

In include/deal.II/fe/mapping_fe.h
<https://github.com/dealii/dealii/pull/727#discussion_r27796491>:

> @@ -0,0 +1,741 @@
> +// ---------------------------------------------------------------------
> +//
> +// Copyright (C) 2001 - 2015 by the deal.II authors

This isn't true. The correct copyright years here is just 2015.

—
Reply to this email directly or view it on GitHub
<https://github.com/dealii/dealii/pull/727/files#r27796491>.